### PR TITLE
Adding A  Fixed  The Navagation  Bar

### PR DESCRIPTION
**Pull request by HeDevedUp**
displaying   navigation bar properly

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+</head>
+
+<body>
+    <header class="container">
+
+        <img class="img_2" src="/asset/image_geometry_1.png" alt="">
+        <img class="img_1" src="/asset/ic_menu.png" alt="">
+
+
+    </header>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] kindly be noticed that the header tag should have the following.
- The logo (if the design has one) is on the left and has a link (a tag).
- The hamburger button is on the right and has a `nav` tag.

- [ ] Also be noticed that the content of the header is aligned with the flexbox.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <p class="content_detail">
+            Hello! I am a software developer! I can help you build a product,
+            feature or website. Taka a look of myworks.
+            If you like what you see and have a project you need coded,
+            don’t hesitate and contact me cheers.
+        </p>
+    </div>
+    <div class="trial">
+        <button class="content_button">Start Collabrating </button>
+    </div>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly wrap all of this inside a `section` tag.
- Also be noticed that you should add the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag) inside the `section` tag.

- [ ] Also be noticed that the content of the section is aligned with flexbox

File: index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    <div class="trial2">
+        <img class="img_3" src="./asset/image_geometry_2.png" alt="">
+    </div>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly be noticed that the section has the proper background image (decorations are always background images, not `img` tags)

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @officailesse , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by JuliCarracedo — APPROVED** [review]
# Status: Approved✔️
Hi @officailesse , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**